content/Simple dynamic settings for Django - Jorge Michelena.md

```diff
@@ -0,0 +1,101 @@
+## Simple dynamic settings for Django
+
+
+### Introduction
+
+Not long ago I finished working on a small project in Django when I realized that it could be improved greatly by adding a single feature.
+There were a few functionalities depending directly on constant values defined in the *settings* file but given their nature it seemed like a good idea to be able to modify them on runtime, that way the application's behavior could be altered without having to edit its settings file and deploy the project again.
+
+After doing some research on the topic I learned that there are a few libraries that provide an implementation to handle editable or dynamic settings, but I felt they are a little overkill for the purposes of this project. Given that, I decided to implement my own solution which is simpler than the ones that are already implemented.
+
+
+### Editing periodic celery tasks schedules
+
+The project included a couple periodic tasks running with Celery and I wanted to be able to edit those tasks' schedules, in this case the solution was very straightforward thanks to [django-celery-beat](https://github.com/celery/django-celery-beat).
+
+Django-celery-beat is a library for Django that provides us with models for periodic tasks and models that help us to define and modify when and how those tasks will be executed; having a great degree of freedom. It can be done through code or the Django admin page, which results very helpful.
```

Comment:
```suggestion
Django-celery-beat is a library for Django that provides us with models for periodic tasks and models that help us to define and modify when and how those tasks will be executed; having a great degree of freedom. It can be done through code or the Django admin page, which is very helpful.
```